docs/try-flink/table_api.zh.md

```diff
@@ -451,6 +457,7 @@ import org.apache.flink.streaming.api.TimeCharacteristic;
 import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
 import org.apache.flink.table.api.Tumble;
 import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+import org.apache.flink.table.api.TableResult;
```

Comment:
use a `org.apache.flink.table.api.*` for all table examples to simplify the imports